app/tests/e2e.test.ts

```diff
@@ -0,0 +1,141 @@
+import fetch from "node-fetch";
+
+type HttpMethod = "post" | "put" | "get" | "patch" | "delete";
+const baseUrl = "http://localhost:5001/helloworld";
+const testId = "test-id-for-raml";
+const transformedTestId = "test-with-transformer";
```

Comment:
How can you ensure this is consistent with transformers file. I.e. if it gets changed in the transformers file, how do we know to change this variable without running the test?

Reply:
exported the common consts to test utils